### DISCUSSION TITLE
Moddable support coexistence

### DISCUSSION THREAD

**Discussion by SanjayEIC** [Moddable-OpenSource/moddable#1284] [Q&A]
Hi am working ESP32 module
1) Is moddable support coexistence (WiFi, BLE both working).

**Comment by phoddie** [comment]
He's, BLE and Wi-Fi can operate simultaneously.

**Reply by SanjayEIC** [reply]
Thanks , Any example support, we will check the coexistence.

**Reply by phoddie** [reply]
There are separate examples for both BLE and Wi-Fi. You can merge two of those to test.

**Reply by SanjayEIC** [reply]
Ok, thanks for support.
